Content.Server/Bible/Components/BibleUserComponent.cs

```diff
@@ -0,0 +1,12 @@
+using Robust.Shared.GameObjects;
+using Robust.Shared.Analyzers;
+
+namespace Content.Server.Bible.Components
+{
+    [RegisterComponent]
+
+    public class BibleUserComponent : Component
+    {
+        public override string Name => "BibleUser";
+    }
```

Comment:
```suggestion
    [RegisterComponent, ComponentProtoName("BibleUser")]
    public sealed class BibleUserComponent : Component {}
```

Technically comp names are automatic based on whatever prefixes component but I just use the attribute regardless.